lib/fibers_stack/SFibersStackHitFinderPar.cc

```diff
@@ -32,6 +32,62 @@ void SFibersStackHitFinderPar::clear()
 {
     fA0 = 0.0;
     fLambda = 0.0;
+    fResPos.Reset();
+    fResEne.Reset();
+}
+/**
+ * The resolution equations are typically f(x) = p0/x + p1/x^{1/2} + p2/x^{3/2}
+ * from Poisson statistics.
+ * Here we can set/get the number of resolution parameters (default=3) and the
+ * parameters themselves
+ */
+UInt_t SFibersStackHitFinderPar::getNumOfResPars() const
```

Comment:
Not needed, as the TArrayF already holds the size in parameters and reading out parameters will check that.